src/databricks/sql/utils.py

```diff
@@ -470,3 +468,33 @@ def _create_arrow_array(t_col_value_wrapper, arrow_type):
             result[i] = None
 
     return pyarrow.array(result, type=arrow_type)
+
+
+class ParamConverter:
+
+    @staticmethod
+    def convert_to_spark_parameters(parameters: Dict[str, Any]) -> List[TSparkParameter]:
+        spark_params = []
+        for k, v in parameters.items():
+            sql_type, spark_value = ParamConverter.get_type_and_value(v)
+            spark_params.append(
+                TSparkParameter(name=k, type=sql_type, value=spark_value)
+            )
+        return spark_params
+
+    @staticmethod
+    def get_type_and_value(value: Any) -> (str, TSparkParameterValue):
```

Comment:
We need to have more comprehensive handling of types.  There needs to be a way for a user to specify the desired sql type of the parameter value.